static/script/devices/mediaplayer/html5.js

```diff
@@ -587,7 +587,7 @@ require.def(
                         self._mediaElement.currentTime = self._sentinelSeekTime;
                     });
                 } else {
-                    this._sentinelSeekTime = currentTime;
+                    this._sentinelSeekTime = undefined;
```

Comment:
Hm, the previous behaviour here of setting _sentinelSeekTime to the current time keeps the sentinel active, but also keeps the target time 'rolling forward' as playback continues.

Setting _sentinelSeekTime to undefined will effectively disable the seek sentinel forever (well, at least until the next playFrom); see line 577. So, doesn't this change mean that the seek sentinel will never be able to work on any device in any circumstances? :-)